### PR TITLE
French translation cleanup

### PR DESCRIPTION
**Pull request by Aldarone**
- Order translations by keys, like in the english template.
- Add some missing gender neutral formulations.

### CODE REVIEW

File: app/assets/javascripts/components/locales/fr.jsx [hunk 1]
```diff
@@ -1,3 +1,11 @@
+/**
+ * Note aux contributeurs⋅trices:
+ * Pour rendre la vie des autres personnes apportant leur contribution,
```
Comment (SansPseudoFix) [inline app/assets/javascripts/components/locales/fr.jsx]: I think you forgot words?! :P

Comment (Aldarone) [inline app/assets/javascripts/components/locales/fr.jsx]: Whoops

File: app/assets/javascripts/components/locales/fr.jsx [hunk 2]
```diff
@@ -1,147 +1,155 @@
+/**
+ * Note aux contributeurs⋅trices:
+ * Pour rendre plus simple la vie des autres personnes
+ * apportant leur contribution, merci de penser aux choses suivantes :
+ *   1. Ajoutez les nouvelles chaînes traduites par ordre alphabétique
+ *   2. Pensez à supprimer les chaîne inutilisées
```
Comment (SansPseudoFix) [inline app/assets/javascripts/components/locales/fr.jsx]: chaîne**s**

Comment (Aldarone) [inline app/assets/javascripts/components/locales/fr.jsx]: Décidément…

**Review by SansPseudoFix — APPROVED** [review]

**Review by Gargron — APPROVED** [review]

**Pull request merged**